croncatd/src/opts.rs

```diff
@@ -19,7 +20,8 @@ pub struct Opts {
     pub cmd: Command,
 }
 
-#[derive(Debug, StructOpt, Clone)]
+#[derive(Debug, StructOpt, Clone, EnumDisplay)]
+#[enum_display(case = "Kebab")]
```

Comment:
Which cases it supports beside Kebab?

Reply:
Any case from https://docs.rs/convert_case/latest/convert_case/